docs/pages/zero-trust-access/zero-trust-access.mdx

```diff
@@ -1,6 +1,286 @@
+import windowsDesktopsSvg from "@site/src/components/Icon/teleport-svg/windows-desktops.svg";
+import autoDiscoverySvg from "@site/src/components/Icon/teleport-svg/auto-discovery.svg";
+import cloudProvidersSvg from "@site/src/components/Icon/teleport-svg/cloud-providers.svg";
+import mcpAndAiSvg from "@site/src/components/Icon/teleport-svg/mcp-and-ai.svg";
+
+<LandingHero
+  title="Zero Trust Access"
+  image={zeroTrustAccessImg}
+>
+  Zero Trust Access provides engineers with just-in-time, least‐privileged
+  access to applications, servers, databases, Kubernetes clusters, and other
+  resources across distributed infrastructures, improving engineering time to
+  market and infrastructure resiliency.
+</LandingHero>
+
+<UseCasesList
+  title="Popular Actions"
+  desktopColumnsCount={2}
+  useCases={[
+    {
+      title: "Sign In with Your Identity Provider",
+      description: "Log in to servers, Kubernetes clusters, databases, web applications, and Windows desktops through their organization's Single Sign-On (SSO) provider.",
+      href: "./sso/",
+      tags: [
+        {
+          name: "Okta",
+          href: "./sso/okta/",
+          icon: "okta",
+        },
+        {
+          name: "GitHub",
+          href: "./sso/github-sso/",
+          icon: "github",
+        },
+        {
+          name: "Entra ID",
+          href: "./sso/entra-id-oidc/",
+          icon: "entraId",
+        },
+        {
+          name: "G Suite",
+          href: "./sso/google-workspace/",
+          icon: "googleCloud",
+        },
+       {
+          name: "more",
+          href: "./sso/",
+          arrow: true,
+        },
+      ]
+    },  
+    {
+      title: "Manage and Track Teleport Audit Events",
+      description: "View audit log generated when users and services interact with your cluster. And export events to store them in a log management platform.",
+      href: "./export-audit-events/",
+      tags: [
+        {
+          name: "Elastic Stack",
+          href: "./export-audit-events/elastic-stack/",
+          icon: "elasticStack",
+        },
+        {
+          name: "Panther",
+          href: "./export-audit-events/panther/",
+          icon: "panther",
+        },
+        {
+          name: "Datadog",
+          href: "./export-audit-events/datadog/",
+          icon: "datadog",
+        },
+        {
+          name: "Splunk",
+          href: "./export-audit-events/splunk/",
+        },
+      ]
+    },
+    {
+      title: "Self-Hosting Teleport",
+      description: "Run Teleport Enterprise in your own infrastructure from setting up high availability and multi-region clusters to secure CA keys with KMS or HSM.",
+      href: "./deploy-a-cluster/",
+      tags: [
+        {
+          name: "AWS KMS",
+          href: "./deploy-a-cluster/aws-kms/",
+          icon: "aws",
+        },
+        {
+          name: "Google Cloud KMS",
+          href: "./deploy-a-cluster/gcp-kms/",
+          icon: "gSuite",
```

Comment:
The "G Suite" (which I think should be updated to reflect it's new name, Google Workspace..see above comment) link above uses a `googleCloud` icon, yet the Google Cloud KMS link here uses a `gSuite` icon. Any particular reason for the difference?